front-end/src/Pages/Client/Client-Product/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { getApiData } from '../../../Redux/action/apiProductsAction';
+import ProductCard from './innerPage/ProductCard';
+import './styles.css';
+
+const ClientProduct = () => {
+  const products = useSelector(state => state.apiProductsReducer.data);
+  const shoppingList = useSelector(state => state.shoppingListReducer.data);
+  const dispatch = useDispatch();
+  
+  const totalPrice = shoppingList.reduce((acc, { price, sellingQnt }) => {
+    return acc + price * sellingQnt
+  }, 0);
```

Comment:
Não ficaria melhor em uma linha??

```suggestion
  const totalPrice = shoppingList.reduce((acc, { price, sellingQnt }) => acc + price * sellingQnt, 0);
```